### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by Marra2342**
Added basic files needed for the exercise. Index.html, style.css, a valid README.md and other files alongside the linters.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.header {
+  color: forestgreen;
```
Comment (Kakalanp) [inline style.css:2]: [OPTIONAL]

Sometimes using names for colors is considered bad practice, you could change it to something like: `color: #228b22;`

**Review by Kakalanp — CHANGES_REQUESTED** [review]
# Hello there @Marra2342! hi

### Good job so far! :tada: 

There are some issues that you still need to work on to go to the next project but you are almost there! :pinching_hand: 

so close

## Highlights :+1: 
- No linter errors :heavy_check_mark:
- Solid pull request :scroll:
- Espectacular first try! :monocle_face: 

## Required Changes ♻️

Great job using the README template! But don't forget to tweak it for each project, adding the correct info to it, this will make your repo more professional :briefcase: 
image


## Optional suggestions

It seems that you have accidentally added [an empty file](https://github.com/Marra2342/Hello-word/blob/30c3a10024f968f6e0117dd56ac6515956c5a317/Hello-World) in your repo, kindly consider deleting it :wink:  

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (Marra2342) [reply style.css:2]: Thank you, will change it.

File: README.md [hunk 2]
```diff
@@ -1,28 +1,30 @@
...
+### To Access my project locally you can follow these steps
 
-To get a local copy up and running follow these simple example steps.
+1. Enter this url: <(https://github.com/Marra2342/Hello-word/pull/1)> in your web browser.
+2. Once opened navigate to the top left level of the project a green code download button will be visible on the righthand side.
+3. Select download Zip option from drop down menu.
+4. Once the download is complete you will be able to access my project locally.
+   Here is my repository link: <(https://github.com/Marra2342/Hello-word/pull/1)>
 
 ### Prerequisites
```
Comment (Ismailco) [inline README.md:29]: - [ ] [Optional] You can remove this section if it's empty and not used, or add some information inside it

File: README.md [hunk 3]
```diff
@@ -1,28 +1,30 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
```
Comment (Ismailco) [inline README.md:1]: - [ ] [OPTIONAL] The Pull Request should have a descriptive title telling you what changes are inside, now the title ```Feature``` is not descriptive, please change it to something meaningful

**Review by Ismailco — APPROVED** [review]
### Hi @Marra2342,
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**